helm/rook-operator/charts/monitoring/templates/ceph-osd-rules.yaml

```diff
@@ -0,0 +1,96 @@
+{{- if .Values.global.rookEnabled }}
+{{- if .Values.global.isManagementCluster }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ .Values.nameStub }}-ceph-osd.rules
+  namespace: {{ .Values.rules.deploymentNamespace }}
+spec:
+  groups:
+  - name: ceph-osd.rules
+    rules:
+    - alert: CephOSDNearFull
```

Comment:
I think it's worth retaining the OSD full alerts as OSDs can fill up even if the cluster isn't full - CRUSH isn't always great at balancing data.